### PR TITLE
chore: [CHAOS-2338]: Added docs for Chaos Module release notes - 1.17.3

### PR DESCRIPTION
**Pull request by Jonsy13**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/chaos-engineering.md [hunk 1]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
   <TabItem value="Early access">
 
 This release does not include early access features.
 
-
   </TabItem>
   <TabItem value="Fixed issues"> 
 
-* There was an issue where users were not getting audit events for the rules created under the Security Governance tab. This issue has been fixed. (CHAOS-2259)
+* Fixed revert chaos for Node drain chaos fault in case of Abort or Chaos Injection failure. (CHAOS-2184)
```
Comment (deepakbolangady) [inline release-notes/chaos-engineering.md:53]: Not clear. Can we write it this way?
```suggestion
* Fixed how chaos is reverted if an attempt to inject the node drain fault fails or needs to be canceled. (CHAOS-2184)
```
The [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/abort-abortion) says "Use abort for a technical audience only if it's part of the application programming interface (API). Use an alternative term in comments and descriptions." It lists some alternative terms for comments and descriptions. Can we use cancel instead? At least in descriptive documentation like this one?

**Review by deepakbolangady — CHANGES_REQUESTED** [review]
I've left a few comments for the Chaos team to consider before I merge.

File: release-notes/chaos-engineering.md [hunk 2]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
-## Latest - August 9, 2023, version 1.16.6
+## Latest - August 21, 2023, version 1.17.3
 
 <Tabs>
   <TabItem value="What's new">
 
-This release does not include new features.
+* Added support for CSMs to be able to view the Activity and Usage for Chaos Module. (CHAOS-1978)
+
+* Added support for Openshift configuration for deploying chaos infrastructure. This will provide users with a predefined SCC & will allow users to modify the same according to their needs. (CHAOS-1889)
```
[the remark this answers is not in the document]
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

File: release-notes/chaos-engineering.md [hunk 3]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Enhanced the Chaos Experiment execution diagram to not switch to running nodes automatically. This will allow users to stay on any node by clicking on the same & observe the respective node details. (CHAOS-2258)
```
[the remark this answers is not in the document]
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

File: release-notes/chaos-engineering.md [hunk 4]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Enhanced docker service kill fault to support containerd runtime. (CHAOS-2220)
```
[the remark this answers is not in the document]
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

File: release-notes/chaos-engineering.md [hunk 5]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Added support for targetting applications by only appkind, only applabel and set-based labels. (CHAOS-2170) and (CHAOS-2128)
```
[the remark this answers is not in the document]
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

File: release-notes/chaos-engineering.md [hunk 6]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Parallel chaos injection/revert operations at scale have been improved for multiple target pods on the same node. (CHAOS-1563)
```
[the remark this answers is not in the document]
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

File: release-notes/chaos-engineering.md [hunk 7]
```diff
@@ -18,24 +18,39 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Updated the behaviour of TARGET_CONTAINER configuration. when not set, the fault will target all containers in target pods. Previously, it used to target only one container randomly. (CHAOS-1216)
```
[the remark this answers is not in the document]
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

File: release-notes/chaos-engineering.md (hunk 1 above)
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Done!

**Review by deepakbolangady — APPROVED** [review]

**Pull request merged**